applications/crossbar/priv/couchdb/swagger/swagger.json

```diff
@@ -8299,6 +8321,26 @@
             "delete": {},
             "get": {}
         },
+        "/v2/accounts/{ACCOUNT_ID}/tasks": {
+            "get": {},
+            "put": {}
+        },
+        "/v2/accounts/{ACCOUNT_ID}/tasks/help": {
+            "get": {}
+        },
+        "/v2/accounts/{ACCOUNT_ID}/tasks/{TASK_ID}": {
+            "delete": {},
+            "get": {},
+            "patch": {}
+        },
+        "/v2/accounts/{ACCOUNT_ID}/tasks/{TASK_ID}/csv": {
+            "put": {}
+        },
+        "/v2/accounts/{ACCOUNT_ID}/tasks/{TASK_ID}/csv/{ATTACHMENT_ID}": {
```

Comment:
Yes, swagger generation appends in case manual changes have occurred.

Pierre Fenoll writes:

>> +        "/v2/accounts/{ACCOUNT_ID}/tasks": {
>> +            "get": {},
>> +            "put": {}
>> +        },
>> +        "/v2/accounts/{ACCOUNT_ID}/tasks/help": {
>> +            "get": {}
>> +        },
>> +        "/v2/accounts/{ACCOUNT_ID}/tasks/{TASK_ID}": {
>> +            "delete": {},
>> +            "get": {},
>> +            "patch": {}
>> +        },
>> +        "/v2/accounts/{ACCOUNT_ID}/tasks/{TASK_ID}/csv": {
>> +            "put": {}
>> +        },
>> +        "/v2/accounts/{ACCOUNT_ID}/tasks/{TASK_ID}/csv/{ATTACHMENT_ID}": {
>
> Oh that shouldn't be there. @jamesaimonetti the generation of this files seems to be appending things? These endpoints are not available anymore
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/2600hz/kazoo/pull/2030/files/7a0ff37f9ac5bc10e57e124d7f671251fb4b98d0#r66123643


--
James Aimonetti

Lead Systems Architect
"If Dialyzer don't care, I don't care"
2600HzPDX | http://2600hz.com
sip:james@2600hz.com
tel:415.886.7905
irc:mc_ @ freenode